ei-distributed-transactions/README.md

```diff
@@ -0,0 +1,63 @@
+##EI Distributed Transactions setup
+
+This docker setup demonstrates performing distributed transactions in EI. The demo setup covers following aspects.
+
+- Transaction mediator performing distributed transactions for a database resource and two JMS resources.
+-  Rollback XA transactions on fault occurences.
+-  Final response to client includes a URL, which can be invoked to retrieve the current transaction status ranging from "Begin"/"End"/"Fault".
+
+##Versions
+
+* [**v1.0.0**](v1.0.0) - Setup using Mysql 5.7.21,ActiveMQ 5.8.0 and WSO2ei- 6.1.1
+
+##How to run
```

Comment:
The heading doesn't render properly in the readme page. Need to add a space after ## symbol.

Reply:
Fixed with https://github.com/wso2/platform-scenarios/pull/11/commits/8cd0126e6cd659415fdf1de129e72841e1006d26